File: dotnetcore_cs/Models/Heat.cs
using System;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
    
namespace App.Models
{  
    public class Heat  
    {  
        [BsonId]
        public ObjectId Id { get; set; }
        [BsonElement("user")]
        public string User { get; set; }
        [BsonElement("x")]
        public int X { get; set; }
        [BsonElement("y")]
        public int Y { get; set; }
    }  
}  

File: php72/crud/app/Models/Heat.php
<?php

namespace App\Models;

use Jenssegers\Mongodb\Eloquent\Model as Eloquent;

class Heat extends Eloquent
{
	protected $connection = 'mongodb';
    protected $collection = 'usersheats';
    protected $primaryKey = '_id';
    protected $fillable = [
        '_id',
        'user',
        'x',
        'y'
    ];
}